src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -25,27 +29,42 @@ namespace OpenTelemetry.Exporter.OpenTelemetryProtocol
     /// </summary>
     public class OtlpExporterOptions
     {
+#if NETSTANDARD2_1
+        /// <summary>
+        /// Gets or sets the target to which the exporter is going to send traces or metrics.
+        /// The valid syntax is described at https://github.com/grpc/grpc/blob/master/doc/naming.md.
+        /// </summary>
+        public Uri Endpoint { get; set; } = new Uri("http://localhost:55680");
+#else
         /// <summary>
         /// Gets or sets the target to which the exporter is going to send traces or metrics.
         /// The valid syntax is described at https://github.com/grpc/grpc/blob/master/doc/naming.md.
         /// </summary>
         public string Endpoint { get; set; } = "localhost:55680";
+#endif
 
+#if NETSTANDARD2_1
+        /// <summary>
+        /// Gets or sets the gRPC channel options.
+        /// </summary>
+        public GrpcChannelOptions GrpcChannelOptions { get; set; }
```

Comment:
The `GrpcChannelOptions` class in `Grpc.Net.Client` should cover functionality previously covered with the `Credentials` and `ChannelOptions` configuration options.